libconsensus/ConsensusEngineBase.h

```diff
@@ -183,6 +184,20 @@ class ConsensusEngineBase : public Worker, virtual public ConsensusInterface
 
     KeyPair const& keyPair() { return m_keyPair; }
 
+    int64_t getAlignedTime() override
+    {
+        if (m_nodeTimeMaintenance)
+        {
+            return m_nodeTimeMaintenance->getAlignedTime();
+        }
+        return utcTime();

```

Comment:
if m_nodeTimeMaintenance is not set, return utcTime acceptable?

Reply:
```
 void resetCurrentTime()
    {
        int64_t parentTime =
            m_blockChain->getBlockByNumber(m_blockChain->number())->header().timestamp();
        m_sealing.block->header().setTimestamp(
            std::max(parentTime + 1, m_consensusEngine->getAlignedTime()));
    }

```

the origin logic before introduce blockchain time align is return `utcTime()`, when `supported_version < v2.6.0`, reset the timestamp of the new block as the previous logic is acceptable.